Example/Backpack/UIKit/Components/Card/Card Button/CardButtonsViewController.swift

```diff
@@ -0,0 +1,280 @@
+                CardButtonSample(
+                    icon: .saved,
+                    backgroundColor: BPKColor.black,
+                    size: .default,
+                    style: .onDark)
+            ]
+        ),
+        CardButtonSampleRow(
+            title: "Size: small. Checked: true",
+            cardButtons: [
+                CardButtonSample(
+                    icon: .saved,
+                    backgroundColor: BPKColor.surfaceDefaultColor,
+                    size: .small,
+                    style: .default),
+                CardButtonSample(
+                    icon: .saved,
+                    backgroundColor: BPKColor.surfaceHighlightColor,
+                    size: .small,
+                    style: .contained),
+                CardButtonSample(
+                    icon: .saved,
+                    backgroundColor: BPKColor.black,
+                    size: .small,
+                    style: .onDark)
+            ]
+        )
+    ]
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = BPKColor.canvasColor
+        title = "Card Button"
+
+        setupView()
+    }
+
+    private func setupView() {
+        let stack = UIStackView(frame: .zero)
+        stack.alignment = .center
+        stack.spacing = BPKSpacingSm
+        stack.translatesAutoresizingMaskIntoConstraints = false
+        stack.axis = .vertical
+
+        let titleStack = UIStackView()
+        titleStack.spacing = BPKSpacingBase
+        titleStack.distribution = .equalCentering
+        ["default", "contained", "onDark"].forEach { text in
+            let styleLabelDefault = BPKLabel()
+            styleLabelDefault.text = text
+            titleStack.addArrangedSubview(styleLabelDefault)
+        }
+        stack.addArrangedSubview(titleStack)
+
+        sampleButtons.forEach { row in
+            let label = BPKLabel()
+            label.numberOfLines = 0
+            label.text = row.title
+            stack.addArrangedSubview(label)
+
+            let horizontalStack = UIStackView()
+            horizontalStack.distribution = .equalCentering
+            row.cardButtons.forEach {
+                let cardButton = makeCardButtonWithContainer($0)
+                horizontalStack.addArrangedSubview(cardButton)
+            }
+            stack.addArrangedSubview(horizontalStack)
+        }
+
+        view.addSubview(stack)
+        NSLayoutConstraint.activate([
+            stack.topAnchor.constraint(
+                equalTo: view.safeAreaLayoutGuide.topAnchor
+            ),
+            stack.leadingAnchor.constraint(
+                equalTo: view.safeAreaLayoutGuide.leadingAnchor
+            ),
+            stack.trailingAnchor.constraint(
+                equalTo: view.safeAreaLayoutGuide.trailingAnchor
+            ),
+            stack.bottomAnchor.constraint(
+                lessThanOrEqualTo: view.safeAreaLayoutGuide.bottomAnchor
+            )
```

Comment:
can you put this in a `setupConstraints` method?

Reply:
13fd5bfed2e7093e80ce6e150a0d38f827068c68